src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkImageBrowser/__tests__/MyCollectionArtworkImageBrowser.jest.tsx

```diff
@@ -0,0 +1,75 @@
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { graphql } from "relay-runtime"
+import { MyCollectionArtworkImageBrowserFragmentContainer } from "../MyCollectionArtworkImageBrowser"
+import { MyCollectionArtworkImageBrowserTestQuery } from "__generated__/MyCollectionArtworkImageBrowserTestQuery.graphql"
+import { screen } from "@testing-library/react"
+import { MockBoot } from "DevTools"
+
+jest.unmock("react-relay")
+
+describe("MyCollectionArtworkImageBrowser", () => {
+  const { renderWithRelay } = setupTestWrapperTL<
+    MyCollectionArtworkImageBrowserTestQuery
+  >({
+    Component: props => {
+      return (
+        <MockBoot>
+          <MyCollectionArtworkImageBrowserFragmentContainer
+            {...(props as any)}
+          />
+        </MockBoot>
+      )
+    },
+    query: graphql`
+      query MyCollectionArtworkImageBrowserTestQuery @relay_test_operation {
+        artwork(id: "artwork-id") {
+          ...MyCollectionArtworkImageBrowser_artwork
+        }
+      }
+    `,
+  })
+
+  describe("when there are images", () => {
+    it("renders correctly", () => {
+      renderWithRelay(
+        {
+          Artwork: () => ({
+            images: [{ width: 800, height: 600 }],
+            figures: [{ __typename: "Image" }],
+          }),
+          Image: () => ({ isDefault: true }),
+          ResizedImageUrl: () => ({
+            width: 800,
+            height: 600,
+            src: "image.jpg",
+            srcSet: "image.jpg 1x",
+          }),
+        },
+        false
+      )
+
+      expect(screen.getByTestId("artwork-lightbox-box")).toHaveStyle(
```

Comment:
Some are of the opinion that using `getByTestId` should be avoided, but I personally very much appreciate the unambiguous simplicity of it and try to encourage its use (over other more general methods)